zanata-common-api/src/main/java/org/zanata/rest/dto/resource/TextFlowTarget.java

```diff
@@ -50,6 +50,14 @@ public Person getTranslator() {
         return translator;
     }
 
+    @Deprecated
+    /**
+     * Zanata server will ignore this field if set,
```

Comment:
I did not remember this being part of the criteria we decided, so I looked for the criteria and could not find any. Did we really decide to make the translator field not work?

@aeng What criteria are you using to implement this?
@djansen-redhat What criteria will you be using to verify this?